.github/workflows/ci.yml

```diff
@@ -81,7 +92,7 @@ jobs:
       - name: Tests
         if: ${{ !cancelled() && steps.deps.conclusion == 'success' }}
         run: |
-          coverage run runtests.py --parallel
+          coverage run runtests.py
```

Comment:
TODO: Make sure to revert this change before merging.

Reply:
Let's revert.